backend/src/v5/models/revisions.js

```diff
@@ -25,6 +26,27 @@ const excludeIncomplete = { incomplete: { $exists: false } };
 
 const collectionName = (model) => `${model}.history`;
 
+const findRevisionsByModel = async (teamspace, model, query, projection) => {
+	const revisions = await db.find(teamspace, collectionName(model), query, projection, {});
+
+	if (!revisions || revisions.length === 0) {
+		throw templates.containerNotFound;
```

Comment:
I don't think we want to throw an error, if there's no revision just return an empty array?

Just thinking the use case of when the user created a model but haven't uploaded a revision yet, you don't expect getting the revision listing for that model to throw you an error, it should just give you no revision.